gcp-cloud-run/src/_artifacts.tf

```diff
@@ -1,11 +1,11 @@
 locals {
   dns_name = var.endpoint.enabled ? var.endpoint.zone.name : null
-  endpoint = var.endpoint.enabled ? "${var.endpoint.subdomain}.${local.dns_name}" : module.application_cloud_run.hostname
+  endpoint = var.endpoint.enabled ? "${var.endpoint.subdomain}.${local.dns_name}" : module.application.hostname
 }
 
 resource "massdriver_artifact" "endpoint" {
   field                = "endpoint"
-  provider_resource_id = module.application_cloud_run.id
+  provider_resource_id = module.application.identity
```

Comment:
We actually probably want the `id` field here if its available. I'm assuming the `id` here is the `grn` of the CloudRun instance? We don't want the application identity (GCP Service Account ID).

Reply:
Fixed! Thanks!